lib/api.js

```diff
@@ -20,6 +20,7 @@ function PlexAPI(options, deprecatedPort) {
     this.hostname = hostname;
     this.port = deprecatedPort || opts.port || PLEX_SERVER_PORT;
     this.https = opts.https;
+    this.timeout = opts.timeout || 60000;
```

Comment:
Could we skip the default timeout value of 60 seconds? With that value we're moving in a gray zone introducing a breaking change. Without a default valut, it will behave just as before.